web_tree_image_tooltip/README.rst

```diff
@@ -72,6 +72,7 @@ Contributors
 * Meet Dholakia <m.dholakia.serpentcs@gmail.com>
 * Nikul Chaudhary <nikul.chaudhary.serpentcs@gmail.com>
 * Phuc Tran Thanh <phuc@trobz.com>
+* Tharathip Chaweewongphan <tharathipc@ecosoft.co.th>
```

Comment:
For plain migrations that do not require substantial changes, normally we don't add ourselves to contributors.
Now, I see that this module didn't follow the rule in the past as well. Not blocking...
Just keep it in mind for the future :wink: